src/main/java/uk/gov/ch/controller/update/trusts/CorporateTrusteesController.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.ch.controller.update.trusts;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.TrustDataCountNotFoundException;
+import uk.gov.ch.model.update.trusts.CorporateTrusteeData;
+import uk.gov.ch.service.update.trusts.CorporateTrusteesService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+public class CorporateTrusteesController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            OracleQueryApplication.APPLICATION_NAME_SPACE);
+    @Autowired
+    private CorporateTrusteesService corporateTrusteesService;
+
+    @GetMapping("overseas-entity/trusts/{trustId}/corporate-trustees")
+    public ResponseEntity<List<CorporateTrusteeData>> getCorporateTrusteeData(
+            @PathVariable String trustId) {
+
+        DataMap dataMap = new DataMap.Builder().build();
+        dataMap.getLogMap().put("trust_id", trustId);
```

Comment:
we are supposed to use the builder. I suspect there is no trustId key pre-defined so in this case maybe just have the trust id in the main log message. Which could well be an info one.